### ISSUE TITLE
loginkey not appended to urls for linux install invite

### ISSUE THREAD

**Issue by tyler71** [Ylianst/MeshCentral#2470]
Using meshcentral:0.7.89

This script works fine when loginKey is disabled

This is the script provided
```
(wget "https://domain.tld/meshagents?script=1" --no-check-certificate -O ./meshinstall.sh || wget "https://domain.tld/meshagents?script=1" --no-proxy --no-check-certificate -O ./meshinstall.sh) && chmod 755 ./meshinstall.sh && sudo ./meshinstall.sh https://domain.tld 'somelongkey' 
```
```
--2021-04-05 01:43:11--  https://domain.tld/meshagents?script=1
Resolving domain.tld (domain.tld)... myipv4addr
Connecting to domain.tld (domain.tld)|myipv4addr|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-04-05 01:43:11 ERROR 404: Not Found.

--2021-04-05 01:43:11--  https://domain.tld/meshagents?script=1
Resolving domain.tld (domain.tld)... myipv4addr
Connecting to domain.tld (domain.tld)|myipv4addr|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-04-05 01:43:11 ERROR 404: Not Found.
```

If I put &key=mykey after the ?script for each wget command, it seems to partially work, but I'll get a `Username/Password Authentication Failed.` error. I don't know if it's related.
```
(wget "https://domain.tld/meshagents?script=1&key=mykey"
```

**Comment by Ylianst** [comment]
Thank you for reporting this. I will take a look at it.

**Comment by Ylianst** [comment]
Just submitted a fix for this. The "Add Agent" link in the main site was correct but the invitation page was not. Should be fixed in MeshCentral v0.8.3 when it's published. Let me know if it works and close this issue once it does.

**Comment by tyler71** [comment]
v0.8.3 resolved it.
Thank you!
